.semaphore/semaphore.yml

```diff
@@ -356,6 +372,11 @@ blocks:
     - name: "Node: k8s-test"
       commands:
       - make k8s-test
+    epilogue:
+      always:
+        commands:
+        - test-results publish /code/report/nosetests.xml --name "node-ci" || true
```

Comment:
`/code` will be the path inside the container; you'll need to find the path on the host